es-space-mgmt/main/default/lwc/reservationList/__tests__/reservationList.test.js

```diff
@@ -0,0 +1,231 @@
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('getOpenReservations @wire data', () => {
+        const element = createElement('c-reservation-list', {
+            is: ReservationList
+        });
+        document.body.appendChild(element);
+
+        // Validate if pubsub got registered after connected to the DOM
+        expect(subscribe).toHaveBeenCalled();
+        expect(subscribe.mock.calls[0][1]).toBe(FLOW_STATUS_CHANGE_MC);
+
+        getOpenReservationsAdapter.emit(mockOpenReservationsRecords);
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve().then(() => {
+            // Select elements for validation
+            const reservationTileElements = element.shadowRoot.querySelectorAll(
+                'c-reservation-tile'
+            );
+            expect(reservationTileElements.length).toBe(
+                mockOpenReservationsRecords.length
+            );
+            // check if the text is rendered fine
+            const openReservationParagraphElement = element.shadowRoot.querySelector(
+                'p.slds-text-heading_small'
+            );
+            expect(openReservationParagraphElement.textContent).toBe(
+                'Open Reservations:'
+            );
+        });
+    });
+
+    it('subscribe to flow status change message', () => {
+        const element = createElement('c-reservation-list', {
+            is: ReservationList
+        });
+        document.body.appendChild(element);
+
+        // Simulate pulishing a message using FLOW_STATUS_CHANGE_MC message channel
+        const messagePayload = {
+            flowName: 'spaceDesigner',
+            status: 'FINISHED'
+        };
+        publish(
+            messageContextWireAdapter,
+            FLOW_STATUS_CHANGE_MC,
+            messagePayload
+        );
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve().then(() => {
+            // Select elements for validation
+            const reservationTileElements = element.shadowRoot.querySelectorAll(
+                'c-reservation-tile'
+            );
+            expect(reservationTileElements.length).toBe(0);
+            expect(subscribe).toHaveBeenCalled();
+            expect(subscribe.mock.calls[0][1]).toBe(FLOW_STATUS_CHANGE_MC);
+            // test refresh method has been called as a result of LMS event
+            expect(refreshApex).toHaveBeenCalled();
+        });
+    });
+
+    it('select a reservation tile from the list and test for Tile Selection LMS publish', () => {
```

Comment:
Again, you're testing too many things in this single test. Split that into smaller tests.
See my previous comment on creating separate test suites for @wire and LMS.

Reply:
done